spec/models/comment_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user1) { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Kenya.', posts_counter: 0) }
+  let(:post) do
```

Comment:
Some specs are failing for Comment model. Kindly see the screenshot below for reference.
![comment_error](https://user-images.githubusercontent.com/35504149/177796576-00d75dd0-579f-407d-930d-1889fa421015.png)